bin/satellite-clone

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+from ansible.cli.playbook import PlaybookCLI
+
+
+def yes_or_no(question):
+    while "the answer is invalid":
+        reply = str(raw_input(question + ' (y/n): ')).lower().strip()
+        if reply[0] == 'y':
+            return True
+        if reply[0] == 'n':
+            exit(1)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-v", "--verbose",
+                        action="store_true",
+                        dest="verbose",
+                        default=False,
+                        help="verbose output")
+    parser.add_argument("--start-at-task",
+                        action="store",
+                        dest="start_at_task",
+                        help="Start at a specific task")
+    parser.add_argument("--step",
+                        action="store_true",
+                        dest="step",
+                        default=False,
+                        help="interactive: confirm each task before running")
+    args = parser.parse_args()
+
+    PRODUCTION_INSTALL_PATH = "/usr/share/satellite-clone"
```

Comment:
You could make this a parameter ```--install-path``` which could help with development setups.

Reply:
right now we use `ansible-playbook` to run in development mode, this script is just for the rpm

Reply:
Usually I find that config options are good to have to make things portable/testable but I'll leave it up to you.